### ISSUE TITLE
README not detected

### ISSUE THREAD

**Issue by oliverviljamaa** [npm/registry-issue-archive#289]
Message on npm page:

> Sadly, this package has no readme. Go write one!

npm page:
https://www.npmjs.com/package/webpack-translations-plugin

GitHub repo:
https://github.com/transferwise/webpack-translations-plugin

I published using npm v5.6.0 and after waiting for ~20 minutes tried republishing.
When `npm install`ing the package, the `README.md` is included, but even though the last publish was 11 hours ago, the npm page does not show it.

**Comment by guillaumearm** [comment]
I have exactly the same problem, however README.md is included too.

npm page:
https://www.npmjs.com/package/handle-io

GitHub repo:
https://github.com/guillaumearm/handle-io

latest published README.md file:
https://unpkg.com/handle-io@latest/README.md

**Edit** : Everything looks good for me now, but not for @oliverviljamaa apparently.

**Comment by silkentrance** [comment]
@oliverviljamaa this might have been fixed, looking at the current package info on npmjs.com.
@isaacs what about closing issues once they have been fixed or resolved?

**Comment by circa10a** [comment]
Having the same issue on https://www.npmjs.com/package/easy-soap-request

**Comment by isaacs** [comment]
@circa10a I see the readme there now.  Did you just publish this?  It might be just a delay in the readme getting into our system?

**Comment by circa10a** [comment]
Yeah looks like the sync was a little behind. Thanks for checking!
